lib/Doctrine/ORM/EntityManager.php

```diff
@@ -222,12 +222,8 @@ public function getCache()
     /**
      * {@inheritDoc}
      */
-    public function transactional($func)
+    public function transactional(callable $func)
```

Comment:
Yes, but not for some alternative/weird syntaxes. :) Don't exactly remember, but one example: https://3v4l.org/a9Ae4